README.md

```diff
@@ -1,5 +1,5 @@
 # PcscSample
-Sample application for PC/SC reader on C-One e-ID
+Sample application for PC/SC reader on C-One e-ID and Access-ER.
```

Comment:
And Access-ER with HID Omnikey

Reply:
Pas forcément que celui-là, c'est pour ça que j'ai les paragraphe suivant, il pourra y avoir d'autres types d'Access-ER avec lecture Pcsc (lecteur smart card par exemple)